app/assets/stylesheets/components/select/_select-equity.scss

```diff
@@ -16,9 +15,6 @@
 
     &__content {
         @include breakpoint-down($small) {
```

Comment:
Thought it was best to leave this in as the image was getting too small with just `object-fit: contain` at larger px.